### PR TITLE
ansible install of OVN on v3.11 cluster to bring up networking

### PR DESCRIPTION
**Pull request by pecameron**
Use ansible to install OVN on top of a 3.11 (no cluster network provisioned) install

On any suitable host, clone ovn-kubernetes

cd dist/yaml
edit ovnkube-master.yaml, ovnkube.yaml, sdn-ovs.yaml for the desired
image name.

cd ../ansible
edit hosts for hostname of master

./run-playbook

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@rajatchopra @dcbw @squeed PTAL
This is the last piece of ovn on v3.11
@weliang1 has verified that it works.

**Comment by shettyg** [comment]
Can you please remove "WIP" from commit message?

**Comment by pecameron** [comment]
@shettyg done.

**Comment by pecameron** [comment]
@dcbw I think you need a / before the lgtm for it to count. I use this and brew pulp is very convenient for me. I hope you are willing to leave it in.

**Comment by pecameron** [comment]
@dcbw @rajat @squeed PTAL

File: dist/ansible/ovn-playbook.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  - name: iptables allow 6641
+    iptables:
+      chain: OS_FIREWALL_ALLOW
+      ctstate: NEW
+      destination_port: 6641
+      jump: ACCEPT
+      protocol: tcp
+
+  # iptables -A OS_FIREWALL_ALLOW -p tcp -m state --state NEW -m tcp --dport 6642 -j ACCEPT
+  - name: iptables allow 6641
```
Comment (zhaozhanqi) [inline dist/ansible/ovn-playbook.yaml]: typo, here should be `iptables allow 6642`

Comment (pecameron) [reply dist/ansible/ovn-playbook.yaml]: good spot. fixed.

**Comment by pecameron** [comment]
Is this ready to merge? 
@dcbw @rajatchopra @squeed PTAL

**Comment by squeed** [comment]
LGTM.

File: dist/READMEopenshiftdevpreview.md [hunk 2]
```diff
@@ -26,61 +26,25 @@ openshift_use_openshift_sdn='False'
...
-# iptables -A OS_FIREWALL_ALLOW -p tcp -m state --state NEW -m tcp --dport 6641 -j ACCEPT
-# iptables -A OS_FIREWALL_ALLOW -p tcp -m state --state NEW -m tcp --dport 6642 -j ACCEPT
-```
-
-The needed ovn daemonsets and setup script are in the openvswitch/ovn-kubernetes
-repo on github.
+be no cluster networking.
 
-Clone ovn-kubernetes:
+Clone ovn-kubernete on a convenient host where you can run ansible-playbook:
```
Comment (rajatchopra) [inline dist/READMEopenshiftdevpreview.md]: s/ovn-kubernete/ovn-kubernetes/

**Comment by dcbw** [comment]
LGTM

**Pull request merged**